ibm/service/configurationaggregator/data_source_ibm_config_aggregator_configurations.go

```diff
@@ -27,65 +27,82 @@ func DataSourceIbmConfigAggregatorConfigurations() *schema.Resource {
 		ReadContext: dataSourceIbmConfigAggregatorConfigurationsRead,
 
 		Schema: map[string]*schema.Schema{
-			"config_type": &schema.Schema{
+			"config_type": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The type of resource configuration that are to be retrieved.",
 			},
-			"service_name": &schema.Schema{
+			"service_name": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The name of the IBM Cloud service for which resources are to be retrieved.",
 			},
-			"resource_group_id": &schema.Schema{
+			"resource_group_id": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The resource group id of the resources.",
 			},
-			"location": &schema.Schema{
+			"location": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The location or region in which the resources are created.",
 			},
-			"resource_crn": &schema.Schema{
+			"resource_crn": {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Description: "The crn of the resource.",
 			},
-			"prev": &schema.Schema{
+			"sub_account": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Filter the resource configurations from the specified sub-account in an enterprise hierarchy.",
+			},
+			"access_tags": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Filter the resource configurations attached with the specified access tags.",
+			},
+			"user_tags": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Filter the resource configurations attached with the specified user tags.",
+			},
+			"service_tags": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Filter the resource configurations attached with the specified access tags.",
```

Comment:
access tags and service tags have same description. can you look into it?

Reply:
addressed